provisioning/utils.go

```diff
@@ -48,21 +49,30 @@ func getPodHostIP(podClient coreclientv1.PodsGetter, targetNamespace string) (st
 	return hostIP, err
 }
 
-func GetServerInternalIP(osclient osclientset.Interface) (string, error) {
+func GetServerInternalIP(osclient osclientset.Interface, config *metal3iov1alpha1.ProvisioningSpec, client kubernetes.Interface, targetNamespace string) (string, error) {
```

Comment:
Let's keep the argument order the same as GetIronicIP

Reply:
Good point. Done!